spec/controllers/conversations_controller_spec.rb

```diff
@@ -203,4 +203,22 @@
             expect(convos[1]).to eq({ "title" => "second", "resolution" => "second resolution" })
         end
     end
+    
+    context "#delete_conversation" do
+        before :each do
+            controller.stub(:check_access_token).and_return(true)
+            arena = double "Arena", :user1_id => 1, :user2_id => 2
+            @conversation = double "Conversation", :update_column => nil, :arena => arena
+            Conversation.stub(:find_by_id).and_return @conversation
+        end
+
+        it "should delete" do
```

Comment:
This test should be renamed to something like `it "should not allow users to delete conversations they are not a part of"`. You will also need another test that checks whether users can delete conversations successfully.